src/elm/Page/Dashboard/Community.elm

```diff
@@ -711,6 +728,18 @@ update loggedIn msg model =
             UR.init { model | autoCompleteState = updated }
                 |> UR.addCmd cmd
 
+        PressedEnter val ->
+            case val of
+                True ->
+                    UR.init model
+                        |> UR.addCmd
+                            (Task.succeed ClickedTransfer
+                                |> Task.perform identity
```

Comment:
this snippet still uses `case` instead of `if`. 

You'll need to test it, but looks like you should call `ClickedSendTransfer` instead of `ClickedTransfer`

Reply:
Hey @lucca65 I have found out why it brings up the submit pin form. 
When `ClickedSendTransfer` is called the second case, https://github.com/Pancakem/frontend/blob/enhancement/keyboard-enter-fix/src/elm/Page/Dashboard/Community.elm#L617, evaluates.

Reply:
@lucca65 I haven't found a solution for it yet. The pin is still prompted. Can it be an auth issue? When I input the pin when prompted the transfer s completed.

Reply:
@Pancakem taking a deeper look into this and I noticed that it only happens when we press enter to choose which user we want to transfer to. If we use the mouse instead of pressing enter it don't show this behaviour.

This means that we need to make Auth.elm ignore enter until it is visible.

The way would be to figure out how the Auth module knows if it can show the auth modal or no, and use that same method to register the `PressedEnter` event (I'm talking the one that exists inside `Auth.elm` not the one you implemented here)

Reply:
@Pancakem I've just committed the solution into your branch. After trying to do this I realize that it required more knowledge on the login process, maybe we can take this slower with you, there are plenty of other issues that might require that knowledge too, so we have plenty of opportunities!